8.8/vips.modules

```diff
@@ -522,7 +522,7 @@
     -->
 
   <cmake id="libjpeg-turbo"
-    cmakeargs="-DCMAKE_INSTALL_LIBDIR:PATH=lib -DWITH_TURBOJPEG=OFF -DENABLE_SHARED=ON -DENABLE_STATIC=OFF"
+    cmakeargs="-DCMAKE_INSTALL_LIBDIR:PATH=lib -DWITH_TURBOJPEG=OFF -DWITH_JPEG8=1 -DENABLE_SHARED=ON -DENABLE_STATIC=OFF"
```

Comment:
Could you explain why JPEG8 is necessary? libjpeg-turbo should make a .pc file without this, I think (though I've not double-checked).

Reply:
Oh, good point, this change may not be required to solve this particular problem.

When I was investigating why the test was failing, I looked at what homebrew does - https://github.com/Homebrew/homebrew-core/blob/master/Formula/jpeg-turbo.rb#L20 - and copied the flag over.

Reply:
I removed the DWITH_JPEG8 flag, built new binaries and all the tests continued to pass so have removed that bit from this PR.